notebooks/data-sources/github/thoth_PR_EDA.ipynb

```diff
@@ -0,0 +1,1892 @@
+{
```

Comment:
<p>When we are filtering by stars should we also include repos which have no issues?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aicoe-aiops/ocp-ci-analysis/pull/490/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='23'/>

Reply:
These repos which do not have issues might still have PRs. However, can we assume that if the repo does not have a single issue it is unlikely that there is much external interaction/contribution on that repo. @suppathak @chauhankaranraj what do you think?

Reply:
HI @oindrillac and @chauhankaranraj ,
Just to give some statistics:
- We have 35 repos which does not have issues.
- Out of those 35, we have 7 repo which have some external contribution on PRs
- [test-infra](https://github.com/thoth-station/test-infra/pulls) has 1 external contribution out of 2 PRs.
- [observations](https://github.com/thoth-station/observations/pulls?q=is%3Apr+is%3Aclosed) has 4 external contributions out of 8 PRs.
- [ansible-role-amun](https://github.com/thoth-station/ansible-role-amun/pulls?q=is%3Apr+is%3Aclosed) has 1 external contribution out of 1 PRs.
- [contra-env-infra](https://github.com/thoth-station/contra-env-infra/pulls?q=is%3Apr+is%3Aclosed) has 3 external contributions out of 3 PRs.
- [rapidsai-build](https://github.com/thoth-station/rapidsai-build/pulls?q=is%3Apr+is%3Aclosed) has 1 external contributions out of 2 PRs.
- [s2i-example](https://github.com/thoth-station/s2i-example/pulls?q=is%3Apr+is%3Aclosed) has 14 external contributions out of 22 PRs.
- [s2i-python-container](https://github.com/thoth-station/s2i-python-container/pulls?q=is%3Apr+is%3Aclosed) have 1 external contributions out of 2 PRs.

So, overall we have 25 external contributions. If you think we can include atleast these 7 repos along with the list of repos prepared for data extraction. We can include them.

Reply:
Thanks for digging further into this @suppathak! 🙌  I would atleast include the repos from this which have PRs from more than 1 external contributor.

Reply:
>  among these 35 repos with no issues, are there any that have a lot of stargazers? 
 @chauhankaranraj , For the [talks](https://github.com/thoth-station/talks) repo, it has 5 stargazers. Another one is [s2i-example](https://github.com/thoth-station/s2i-example), it has 3 stargazers. 

>  feel like since there are several external contributors
I think we do have several contributions but not several external contributors.

Reply:
so lets include these 7 repos or atleast contra-env-infra, observations and s2i-example